tests/shared/drive-constraints.spec.js

```diff
@@ -556,4 +556,133 @@ describe('Shared: DriveConstraints', function() {
 
   });
 
+  describe('.getDriveFeedback', function() {
+
+    describe('given there should be no errors or warnings', () => {
+
+      it('should return null', function() {
+        const result = constraints.getDriveFeedback({
+          device: '/dev/disk1',
+          mountpoints: [],
+          name: 'USB Drive',
+          size: 100000000100,
+          protected: false,
+          system: false
+        }, {
+          path: '/mnt/disk2/rpi.img',
+          size: 1000000000
+        });
+
+        m.chai.expect(result).to.equal(null);
+      });
+    });
+
+    describe('given the drive contains the image', () => {
+
+      it('should return the source error', function() {
+        const result = constraints.getDriveFeedback({
+          device: '/dev/disk1',
+          mountpoints: [ {
```

Comment:
I think @jviotti  meant 'on separate lines' rather than 'in separate strings' ;-)